### PR TITLE
UF-9526: Added municipality to API and added it for calls to Document service

### PR DESCRIPTION
**Pull request by MickeWallner**

### CODE REVIEW

File: src/test/java/se/sundsvall/esigning/api/ProcessResourceFailuresTest.java [hunk 1]
```diff
@@ -30,10 +17,24 @@
...
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_PROBLEM_JSON;
+import static org.zalando.problem.Status.BAD_REQUEST;
+
 @SpringBootTest(classes = Application.class, webEnvironment = RANDOM_PORT)
 @ActiveProfiles("junit")
 class ProcessResourceFailuresTest {
```
Comment (foobar-1337) [inline src/test/java/se/sundsvall/esigning/api/ProcessResourceFailuresTest.java:35]: Add test for invalid municipalityId?

File: src/main/java/se/sundsvall/esigning/api/ProcessResource.java [hunk 2]
```diff
@@ -48,9 +50,10 @@ public ProcessResource(ProcessService service) {
 	@ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	@ApiResponse(responseCode = "502", description = "Bad Gateway", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	public ResponseEntity<StartResponse> startProcess(
+		@Parameter(name = "municipalityId", description = "Municipality ID", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
 		@RequestBody @NotNull @Valid SigningRequest request) {
 
-		final var startProcessResponse = new StartResponse(service.startProcess(request));
+		final var startProcessResponse = new StartResponse(service.startProcess(municipalityId, request));
 		LOGGER.info("Request for start of e-signing process for request {} has been received, resulting in an instance with id {}", request, startProcessResponse.getProcessId());
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/esigning/api/ProcessResource.java:57]: Add municipalityId in this log entry?

Comment (AndreSoderlund) [reply src/main/java/se/sundsvall/esigning/api/ProcessResource.java:57]: I did something similar in another application, but SonarCloud is not a fan of logging user input.

Comment (MickeWallner) [reply src/main/java/se/sundsvall/esigning/api/ProcessResource.java]: I have now added it.

**Review by foobar-1337 — APPROVED** [review]

**Pull request merged**